oidc-consumer/DOCUMENTATION.md

```diff
@@ -47,7 +54,7 @@ options to be passed to setup express-sessions
 <a name="OidcConsumer+session"></a>
 
 ### oidcConsumer.session
-session-instance created using config passed from the user
+session-instance created using config passed from the user (read-only)
```

Comment:
is session really read-only? We manipulate it and also destroy it.

Reply:
this property is read-only, the user can only interact w/ it (save/destroy) not override it (consumer.session = customSession)

removed the flag as it wasn't clear @ [1c387c5](https://github.com/getfundwave/idp-consumer/pull/2/commits/1c387c5c8bcf14b3f825a0b004017337b8b9c31d)